code/modules/antagonists/traitor/objectives/eyesnatching.dm

```diff
@@ -75,7 +75,7 @@
 		if(!targets_current.getorgan(/obj/item/organ/internal/eyes))
 			continue
 
-		possible_targets += targets_current
+		possible_targets += possible_target
 
 	for(var/datum/traitor_objective/eyesnatching/objective as anything in possible_duplicates)
 		possible_targets -= objective.victim
```

Comment:
if possible targets is a list of minds now, doesn't that break the use of victim here? should it be victim.mind?

Reply:
if I do that victim_mind gets the value null(I had checked it in a local server)

Reply:
that's ok, it just means they don't have one, right? that's fine for our purposes, so long as if they have one it's removed from the list

Reply:
yeah so what should i change?